Modules/Invoice/Resources/views/subviews/edit/invoice-details.blade.php

```diff
@@ -58,6 +58,15 @@
                         </span>
                     </div>
                 </div>
+                <div class="form-group" v-if="this.client.type == 'indian'">
+                    <div class="d-flex">
+                        <label for="client_id" class="mr-5">total amount:</label>
+                        <span>
+                            {{ $invoice->display_amount . " + " . $invoice->gst ." ₹". " = " }} <span id="totalAmount">{{$invoice->total_amount }}</span><span> ₹ </span>
+                        </span>
```

Comment:
Can we write it like this?
```suggestion
                        <span>
                            {{ $invoice->display_amount . " + " . $invoice->gst ." ₹". " = " }} <span id="totalAmount">{{$invoice->total_amount }} ₹</span>
                        </span>
```